src/Core/CommandBus/Parser/CommandHandlerDefinitionParser.php

```diff
@@ -0,0 +1,204 @@
+    {
+        if (!$constructor = $command->getConstructor()) {
+            return [];
+        }
+
+        $params = [];
+        foreach ($constructor->getParameters() as $parameter) {
+            $param = sprintf('%s', $parameter->getName());
+            if ($type = $parameter->getType()) {
+                /* @phpstan-ignore-next-line ignoring getName() as method is not visible to phpstan */
+                $param = sprintf('%s %s', $type->getName(), $param);
+            }
+
+            if ($parameter->isOptional()) {
+                if ($parameter->allowsNull()) {
+                    $param = sprintf('?%s', $param);
+                }
+                $param = sprintf('%s = %s', $param, var_export($parameter->getDefaultValue(), true));
+            }
+            $params[] = $param;
+        }
+
+        return $params;
+    }
+
+    /**
+     * Parses return type from docblock
+     *
+     * @param ReflectionClass $handlerReflection
+     *
+     * @return string
+     */
+    private function parseReturnType(ReflectionClass $handlerReflection): ?string
+    {
+        $method = $handlerReflection->getMethod(self::HANDLER_METHOD_NAME);
+
+        foreach ($handlerReflection->getInterfaces() as $interface) {
+            if ($interface->hasMethod(self::HANDLER_METHOD_NAME)) {
+                $method = $interface->getMethod(self::HANDLER_METHOD_NAME);
+                break;
+            }
+        }
+
+        if ($returnType = $this->parseReturnTypeFromDocblock($method)) {
+            return $returnType;
+        }
+
+        if ($method->hasReturnType()) {
+            /* @phpstan-ignore-next-line ignoring getName() as method is not visible to phpstan */
+            return $method->getReturnType()->getName();
+        }
+
+        return null;
+    }
+
+    /**
+     * @param ReflectionMethod $method
+     *
+     * @return string|null
+     */
+    private function parseReturnTypeFromDocblock(ReflectionMethod $method): ?string
+    {
+        $docBlock = $method->getDocComment();
+        if (!$docBlock) {
+            return null;
+        }
+
+        $tagPosition = strpos($docBlock, self::RETURN_TAG);
+
+        if (false !== $tagPosition) {
+            $returnType = substr($docBlock, $tagPosition, (strpos($docBlock, PHP_EOL, $tagPosition)) - $tagPosition);
+            $returnType = str_replace(sprintf('%s ', self::RETURN_TAG), '', $returnType);
+
+            return $returnType;
+        }
+
+        return null;
+    }
+
+    /**
+     * @param ReflectionClass $reflectionClass
+     *
+     * @return string
+     */
+    private function parseDescription(ReflectionClass $reflectionClass): string
+    {
+        if (!$docBlock = $reflectionClass->getDocComment()) {
+            return '';
+        }
+
+        /**
+         * Removes comment symbols, annotations, and line breaks.
+         */
+        $description = preg_replace("/\/+\*\*|\*+\/|\*|@(\w+)\b(.*)|\n/",
```

Comment:
Could you create tests for this
because I'm not really sure about your regex.
I don't see the reason of `\/+\*\*` because we never do something like `///////**`.
Could you also please use a different character for delimiters, like `~` or `#`, you will be able to remove unneeded escape.